datachannel_test.go

```diff
@@ -155,6 +155,63 @@ func TestDataChannel_Open(t *testing.T) {
 
 		assert.Len(t, openCalls, 1)
 	})
+
+	t.Run("handler should be called once when already negotiated", func(t *testing.T) {
+		report := test.CheckRoutines(t)
+		defer report()
+
+		offerPC, answerPC, err := newPair()
+		if err != nil {
+			t.Fatalf("Failed to create a PC pair for testing")
+		}
+
+		done := make(chan bool)
+		answerOpenCalls := make(chan bool, 2)
```

Comment:
Why 2? Please use named constant.

Reply:
the existing test does the same thing for the sake of ensuring that an open happens only once but gives it the capacity for two in order to check for that failure.

Reply:
added constants